web/web.go

```diff
@@ -146,9 +146,13 @@ func New(st local.Storage, qe *promql.Engine, rm *rules.Manager, status *Prometh
 	instrf := prometheus.InstrumentHandlerFunc
 	instrh := prometheus.InstrumentHandler
 
-	router.Get("/", instrf("status", h.status))
+	router.Get("/", instrf("graph", h.graph))
```

Comment:
It is not a prometheus debugging page. I frequently go there to interactively explore metrics. Querying is the primary use case for a timeseries database, otherwise you might as well put a status page on `/dev/null`. If the graph page is not obvious to use that is its own problem, but I don't think the status page is any more welcoming to a new user.